test/gui/test_main_window.py

```diff
@@ -197,6 +198,7 @@ def test_on_menu_action_help(default_main_window, monkeypatch):
 def test_on_menu_action_changelog(default_main_window, monkeypatch, has_changelog):
     mock_show = MagicMock()
     monkeypatch.setattr(QtWidgets.QWidget, "show", mock_show)
+    monkeypatch.setattr(ChangeLogWidget, "setup_lables", MagicMock())
```

Comment:
Same function, same typo, just that this one won't get catched with (at least my) automatic refactoring.